Packs/SingleConnect/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "SingleConnect",
+    "name": "Single Connect",
     "description": "Single Connect enables enterprises to remove static passwords stored in applications by instead keeping passwords in a secure password vault. It secures access to passwords through token-based authentication",
     "support": "partner",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
     "author": "",
```

Comment:
Hi @olbelkron 
please add author to the metadata

Reply:
Hi @edik24 
done